components/common/accordion/accordion.tsx

```diff
@@ -12,46 +14,50 @@ export type AccordionItemType = {
     extraContent?: React.ReactNode;
     icon?: React.ReactNode;
     items: React.ReactNode[];
+    id?: string;
 };
 
 type AccordionProps = {
     accordionItemsData: AccordionItemType[];
     expandMultiple?: boolean;
+    openFirstElement?: boolean;
     onHeaderClick?: <T>(ref: RefObject<HTMLLIElement>) => T | void;
+    className?: string;
+    onItemOpened?: (index: number) => void;
 };
 
 const Accordion = ({
     accordionItemsData,
     expandMultiple,
+    openFirstElement,
```

Comment:
question, wouldn't make sense to receive the indexes of the items we would like to be opened on load? in this case you would just send `[0]`

Reply:
That's actually what I did first and I reverted to this because it made more semantic sense? I had my accordion like:

```ts
<Accordion 
   //...
   openOnFirstDraw={[0]}
   //...
/>
```

But realized that most of the time we'll want the first element to be open or everything close by default, and a bool was more elegant than a `[0]` as an argument. I guess I'm 3/5 on that but if there's other use cases in the near future that needs such a change it would make sense to allow it

Reply:
ok, the point is valid, but if you are making the change to the component I think receiving the array instead of a boolean makes more sense, you can always use a semantic constant name like `OPEN_FIRST_ELEMENT = 0; ... [OPEN_FIRST_ELEMENT] ` and will be as explanative as the boolean value. My point is, at the end you are making the same change, one gets limited to the first element the other one is more extendable to open other elements.